docs-app/.prettierrc.js

```diff
@@ -18,5 +15,13 @@ module.exports = {
         singleQuote: false,
       },
     },
+    {
+      files: '*.{cjs,cts,js,mjs,mts,ts}',
+      options: {
+        printWidth: 80,
+        singleQuote: true,
+        trailingComma: 'all',
```

Comment:
In `prettier@v3`, the default value for `trailingComma` will change from `es5` to `all`.